spec/integration/make/dependencies_spec.rb

```diff
@@ -21,18 +19,19 @@
     end
   end
 
+
```

Comment:
```suggestion
```

Reply:
Thanks for spotting, I've amended with that 👍.